.gitignore

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: MIT
+# Copyright (C) 2017 Version Control System Fund
+# Copyright (C) 2021 nfitzen
```

Comment:
Where did these come from? I think if we need to add more copyright information, I would rather write our own gitignore from scratch so we don't need anything :)